util/src/main.ts

```diff
@@ -27,13 +27,23 @@ function readInput(format: string, inputPath: string, callback: (input: LSIF.Ele
 				buffer.push(line);
 				break;
 			case 'line': default:
-				input.push(JSON.parse(line));
+				try {
+					const element: LSIF.Element = JSON.parse(line);
+					input.push(element);
+				} catch {
+					// Do nothing for now
```

Comment:
Is it safe to just ignore this exception.
Users may wonder why the line wasn't included if we just quietly skip it.

Reply:
My intention was to ignore errors if a single line was mistakenly treated as a JSON (in the case the user forgot to specify input format JSON instead of JSON Lines). However, you're right - let's say the user actually wants to pass a JSON Lines input, but one line is wrong. In this case, it will be skipped without any warning.

Do you think a simple console warning is enough in this case? Or should we abort if a subset of lines are broken?

Reply:
I decided to print any lines that fail to parse (regardless of input format) and abort execution. This way, if the user forgot to specify `--inputFormat=json` they will be notified. If however they have a handful of invalid JSON Lines, they will know which ones they are.